resources/aws/asg_stack.go

```diff
@@ -19,6 +20,8 @@ const (
 	defaultTimeout = 5
 	// imageIDParam is the Cloud Formation parameter name for the ASG image ID.
 	imageIDParam = "ImageID"
+
+	stackDoesNotExistError = "Stack with id %s does not exist"
```

Comment:
I think this is fine, but would be good to have an integration test, just to make sure error message didn't changed.